hypha/apply/funds/views.py

```diff
@@ -1230,9 +1235,29 @@ class ReviewerLeaderboardDetail(SingleTableMixin, ListView):
     template_name = 'funds/reviewer_leaderboard_detail.html'
 
     def get_context_data(self, **kwargs):
-        User = get_user_model()
         obj = User.objects.get(pk=self.kwargs.get('pk'))
         return super().get_context_data(object=obj, **kwargs)
 
     def get_table_data(self):
         return super().get_table_data().filter(author__reviewer_id=self.kwargs.get('pk')).select_related('submission')
+
+
+@method_decorator(staff_required, name='dispatch')
+class StaffAssignments(SingleTableMixin, ListView):
+    model = User
+    table_class = StaffAssignmentsTable
+    table_pagination = False
+    template_name = 'funds/staff_assignments.html'
+
+    def get_queryset(self):
+        # Only list staff.
+        return self.model.objects.staff()
+
+    def get_table_data(self):
+        reviewer_roles = ReviewerRole.objects.all().order_by('order')
+        reviewer_role_id_1 = reviewer_roles.first().id
+        reviewer_role_id_2 = reviewer_roles.last().id
+        return super().get_table_data().annotate(
+            role1=Count('assignedreviewers', filter=Q(assignedreviewers__role_id=reviewer_role_id_1)),
+            role2=Count('assignedreviewers', filter=Q(assignedreviewers__role_id=reviewer_role_id_2)),
+        )
```

Comment:
@Parbhat I would like to filter this down to only include assigned reviewers on submissions that are in status "active_statuses". Any tips?

Reply:
@Parbhat Here also I would like to accommodate an arbitrary number of reviewer roles if possible.

Reply:
Similar to the role, we can add a [submission](https://github.com/OpenTechFund/hypha/blob/32679916c2f9e5e82322296b14966659b5d708b9/hypha/apply/funds/models/submissions.py#L967) query to check status is in the active statuses.

Reply:
Looks like we do not need to annotate here as can use `extra_columns` parameter. More details in another comment.